target/linux/ramips/dts/mt7621_samknows_whitebox-v8.dts

```diff
@@ -129,6 +129,7 @@
 		reg = <0x0000 0 0 0 0>;
 		nvmem-cells = <&eeprom_factory_0>;
 		nvmem-cell-names = "eeprom";
+		ieee80211-freq-limit = <2400000 2500000>;
```

Comment:
This is not needed, it uses a MT7603E which only support 2.4GHz.

Reply:
Look at issue https://github.com/openwrt/openwrt/issues/15391

Reply:
@hauke if you think the problem has a deeper origin I can test a alternative fix.